app/src/app/store/mapStore.ts

```diff
@@ -121,6 +127,8 @@ export const useMapStore = create(
     setBrushSize: (size) => set({ brushSize: size }),
     isPainting: false,
     setIsPainting: (isPainting) => set({ isPainting }),
+    paintFunction: getFeaturesInBbox,
+    setPaintFunction: (paintFunction) => set({ paintFunction }),
```

Comment:
Using this pattern made it really easy to handle different behaviors on hover without messing with map events.

Reply:
i like it!